common/common-rest/src/test/java/org/apache/servicecomb/common/rest/codec/param/TestCookieProcessor.java

```diff
@@ -67,8 +71,12 @@ public void testGetValueNoCookies() throws Exception {
     };
 
     CookieProcessor processor = createProcessor("c1", String.class);
-    Object value = processor.getValue(request);
-    Assert.assertNull(value);
+    try {
+      processor.getValue(request);
```

Comment:
This is a behavior change. And it's better to add a new test case which have the same meaning as the old test case. 
CookieProcessor processor = createProcessor("c1", String.class, null, false);